results-processor/wptreport.py

```diff
@@ -552,7 +552,7 @@ def _channel_to_labels(browser: str, channel: str) -> Set[str]:
     if channel == 'release':
         # e.g. Edge release
         labels.add('stable')
-    if channel == 'canary' and browser == 'edgechromium':
+    if channel == 'canary' and (browser == 'edgechromium' or browser == 'edge'):
```

Comment:
I believe the checks failed because this line is 1 character too long. 🙂

Reply:
Fixed!